rclcpp_components/src/component_manager.cpp

```diff
@@ -223,9 +223,6 @@ ComponentManager::create_node_options(const std::shared_ptr<LoadNode::Request> r
             "Extra component argument 'enable_logger_service' must be a boolean");
       }
       options.enable_logger_service(extra_argument.get_value<bool>());
-    } else {
-      throw ComponentManagerException("Extra component argument '" + extra_argument.get_name() +
-          "' is not supported");
```

Comment:
this throw is added to check the unknown extra arguments in the base class.

but i do not think it should throw exception in the base class, instead it should ignore all unknown extra arguments in the base class. exception can only be thrown if the supported argument parameter type is unexpected just like before. so that child class can override this method to add more extra arguments to support. (it can actually try&catch the exception in the child class, but I think throwing exception is not correct here in the 1st place.)

this fixes the issue https://github.com/ros2/domain_bridge/issues/85